tests/sps/validation/test_article_toc_sections.py

```diff
@@ -53,109 +41,240 @@ def test_validate_article_toc_sections_returns_ok(self):
+                'xpath': ".//article-meta//subj-group[@subj-group-type='heading']/subject",
+                'validation_type': 'value in list',
+                'response': 'OK',
+                'expected_value': ['Artículo', 'Editorial', 'Carta'],
+                'got_value': 'Artículo',
+                'message': "Got Artículo expected ['Artículo', 'Editorial', 'Carta']",
+                'advice': None
             }
         ]
-        result_article_sections = self.article_toc_sections.validate_article_toc_sections(expected_section)
+        obtained = self.article_toc_sections.validate_article_toc_sections(expected_section)
 
-        self.assertIn(expected[0], result_article_sections)
-        self.assertIn(expected[1], result_article_sections)
+        for i, item in enumerate(obtained):
+            with self.subTest(i):
+                self.assertDictEqual(expected[i], item)
 
-    def test_validate_article_toc_sections_returns_error(self):
+    def test_validate_article_toc_sections_fail_sections_not_obtained(self):
+        self.maxDiff = None
         self.xmltree = etree.fromstring(
             """
             <article xml:lang="es">
             <front>
                 <article-meta>
                     <title-group>
-                        <article-title>Título do artigo em espanhol</article-title>
-                            <trans-title-group xml:lang="en">
-                                <trans-title>Título do artigo em inglês</trans-title>
-                            </trans-title-group>
+                        <article-title>Título del artículo</article-title>
+                    </title-group>
+                    <article-categories>
+
+                    </article-categories>
+                </article-meta>
+            </front>
+            <sub-article article-type="translation" xml:lang="en">
+                <front-stub>
+                    <article-categories>
+
+                    </article-categories>
+                    <title-group>
+                        <article-title>Article title</article-title>
+                    </title-group>
+                </front-stub>
+            </sub-article>
+            </article>
+            """
+        )
+        self.article_toc_sections = ArticleTocSectionsValidation(self.xmltree)
+        expected_section = {
+                "es": ["Artículo", "Editorial", "Carta"],
+                "en": ["Article", "Editorial", "Letter"]
+            }
+        expected = [
+            {
+                'title': 'Article or sub-article section title validation',
+                'xpath': ".//subj-group[@subj-group-type='heading']/subject",
+                'validation_type': 'exist',
+                'response': 'ERROR',
+                'expected_value': ['Article', 'Editorial', 'Letter'],
+                'got_value': None,
+                'message': "Got None expected ['Article', 'Editorial', 'Letter']",
```

Comment:
Trocar "Got None expected ['Article', 'Editorial', 'Letter']" por "Got None expected one of ['Article', 'Editorial', 'Letter']"

Reply:
OK, @robertatakenaka